### PR TITLE
allow for add_terms to specify organization name

### PR DESCRIPTION
**Pull request by fgregg**
@jpmckinney here's what was in #220 but squashed.

### CODE REVIEW

File: pupa/tests/importers/test_membership_importer.py [hunk 1]
```diff
@@ -89,3 +89,40 @@ def test_no_membership_for_person_including_party():
 
     with pytest.raises(NoMembershipsError):
         memimp.import_data([p._related[0].as_dict()])
+
+
+@pytest.mark.django_db
+def test_multiple_orgs_of_same_class():
+    """
```
Comment (jpmckinney) [inline pupa/tests/importers/test_membership_importer.py]: I'm not sure that this comment applies to this test. Seems to have been copied from `test_no_membership_for_person_including_party`.

**Comment by jpmckinney** [comment]
Once the stray comment is removed, I can merge.

**Comment by fgregg** [comment]
Okay, this should be ready to go.

**Comment by jpmckinney** [comment]
Thanks!

**Pull request merged**